src/coreclr/jit/llvm.cpp

```diff
@@ -2585,10 +2576,14 @@ void Llvm::createAllocasForLocalsWithAddrOp()
 
     for (unsigned lclNum = 0; lclNum < _compiler->lvaCount; lclNum++)
     {
+        if(_compiler->lvaOutgoingArgSpaceVar == lclNum)
```

Comment:
```suggestion
    {
        // TODO-LLVM: Consider turning off FEATURE_FIXED_OUT_ARGS.
        if(_compiler->lvaOutgoingArgSpaceVar == lclNum)
```
(I think the TODO is still relevant -- just the matter is not pressing and merge conflicts are best avoided)